toolchain/check/testdata/packages/fail_conflict_namespace_first.carbon

```diff
@@ -17,6 +15,10 @@ fn NS.Foo() {}
 
 package Example library "fn" api;
 
+// CHECK:STDERR: fn.carbon:[[@LINE+4]]:1: ERROR: Duplicate name being declared in the same scope.
+// CHECK:STDERR: fn NS() {}
+// CHECK:STDERR: ^~~~~~~~~
+// CHECK:STDERR: namespace_then_fn.carbon: Name is previously declared here.
```

Comment:
Do you know why we don't get a line and column number here?

Reply:
Yes, see #3640 (different code, support is already pending)